ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py

```diff
@@ -97,7 +97,7 @@ def process(self, device, results, log):
             pass
 
         # if NTDS service present then this is a DC
-        if domainController:
+        if len(domainController):
```

Comment:
So line 60 can be changed to: domainController = results.get('ActiveDirectory', ())